blog/2021-03-16-BatchSolvingWithAmq.adoc

```diff
@@ -0,0 +1,182 @@
+= Batch solving in a stateless manner with OptaPlanner and AMQ
```

Comment:
The context is OptaPlanner, so AMQ should go before OptaPlanner
"in a stateless manner" => (important) impl detail for later. The title is about motivation, gain, ROI. What is the motivation do this? I'd argue to solve in the cloud? Or to horizontally scale solving datasets?

```suggestion
= Batch solving an AMQ queue full of planning problem datasets
```

Probably not the best suggestion? Any better ideas?

Reply:
How about: "Batch solving an AMQ queue that contains planning problem data sets in a scalable way"? It shows what the motivation is, but I've got a feeling it's starting to be too long.

-1 to talk about cloud in the title as this article doesn't show it.